### DISCUSSION TITLE
🚀 More debug- add logging of validation errors

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3995] [Feature Requests]
### Suggest an idea for ExpressionEngine

I've spent a fair bit of time tracking down different validation errors and often have to hack to see them.  It would be awesome if there was a way to log them- based on a config setting.  Either in the log file or in the database.  DB easier for me, since I don't have to get ftp or have them set it up, but I can see either making sense.  

I hacked it into the entry- just displaying, but would be easy enough to log.  But it feels like it should be in the validation service and log anything coming into that.  And that was a bridge too far 'on the fly'.  Hence- feature request to do it properly.

**Comment by intoeetive** [comment]
Getting this added in https://github.com/ExpressionEngine/ExpressionEngine/commit/430b1b35ed5128c7a09d44497441a54f8cf8b256
